NewSpring/ApollosProject
# (DEV-3698) Create Anonymous attribute value for prayer requests and update resolver

## DESCRIPTION
We had an issue with our current functionality to allow for anonymous prayer requests. We decided to create a new attribute value name `IsAnonymous` to determine if a prayer should display personal info within the app.

### What does this PR do, or why is it needed?
This PR adds a new property to the schema, datasource, and resolver to allow for the creation of new prayers with the `IsAnonymous` attribute. This also allows us to query the value.

### How do I test this PR?
1. Login on the Graphql playground
2. Check to see if there are prayers with the `IsAnonymous` attribute value on Alpha-Rock
3. Run this mutation query to create a new prayer with `IsAnonymous` equaling true:
```
mutation {
  addPublicPrayerRequest(IsPublic: true, FirstName: "Test", LastName: "Bro", Text: "Jesus Rocks", CampusId: 16, CategoryId: 1, IsAnonymous: "True") {
    id
    firstName
    lastName
    text
    requestedByPersonAliasId
    campusId
    categoryId
    flagCount
    prayerCount
    isAnonymous
    person {
      id
      firstName
      lastName
    }
  }
}
```
4. Confirm the newly created mutation has `IsAnonymous` set to true.
---

> I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))

## TODO

- [x] Closes [DEV-3698](https://newspring.atlassian.net/browse/DEV-3698)

## REVIEW

**Manual QA**


**Code Review: Questions to consider**


---

> The purpose of PR Review is to _improve the quality of the software._
